src/commcare_cloud/ansible/roles/commcarehq/tasks/webworkers.yml

```diff
@@ -25,6 +37,7 @@
         no_proxy: "{% if http_proxy_address is defined %}{{ groups['all'] | join(',') }},{{ app_processes_config.additional_no_proxy_hosts }}{% endif %}"
         TMPDIR: '{{ encrypted_tmp }}'
         DATADOG_PATCH_MODULES: "{% if app_processes_config.datadog_pythonagent %}{{ datadog_patch_modules|join(',') }}{% endif %}"
+        prometheus_multiproc_dir: "{% if prometheus_monitoring_enabled|default(False) %}{{ cchq_home}}/www/{{ deploy_env }}/metrics {% endif %}"
```

Comment:
this is also needed for celery and pillowtop processes

Reply:
Added to celery and pillow too